### PR TITLE
Correctly reflect queue maximum size based on namespace SKU

### LINKED ISSUE #499: Premium ServiceBus - Create Queue/Topic with max Queue Size - 80GB
Premium ServiceBus provide facility to create Queue/Topic with max **80GB** Queue Size. 
But ServiceBus explorer provides facility to create queue upto **5GB** only. 

Can you please provide facility to create Queue/Topic with max capacity. 

Note - This is not a priority requirement, but good to have this feature, so we don't need to create Queue from Azure Portal with max size.

image

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #499 

The maximum queue length controller (slider) to reflect the maximum queue size available for the used namespace SKU.

image

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-queue-max-size-to-reflect-namespace-sku.11.build.774 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37147591) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/9c64187885 by @SeanFeldman)

File: src/ServiceBus/Helpers/ServiceBusHelper2.cs [hunk 1]
```diff
@@ -19,17 +19,27 @@
...
 using Azure.Messaging.ServiceBus;
-#endregion
 
 // ReSharper disable CheckNamespace
 namespace ServiceBusExplorer.ServiceBus.Helpers
 // ReSharper restore CheckNamespace
 {
+    using System.Threading.Tasks;
+    using Azure;
+    using Azure.Messaging.ServiceBus.Administration;
```
Comment (ErikMogensen) [inline src/ServiceBus/Helpers/ServiceBusHelper2.cs:30]: Picky, but these usings should be put together with the existing using.

Comment (SeanFeldman) [reply src/ServiceBus/Helpers/ServiceBusHelper2.cs:30]: ok. This should be enforced by the .editorconfig rather than done manually.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-queue-max-size-to-reflect-namespace-sku.12.build.777 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37147698) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/2f66d89779 by @SeanFeldman)

**Review by ErikMogensen — DISMISSED** [review]
Great! 

I just realized that the same change should be done for topics. Will you do that too, @SeanFeldman ? As far as I concerned it may be put In a new PR or in this one.

**Comment by SeanFeldman** [comment]
Good catch. I'll amend this PR tonight.

File: src/ServiceBus/Helpers/ServiceBusHelper2.cs (hunk 1 above)
Comment (ErikMogensen) [reply src/ServiceBus/Helpers/ServiceBusHelper2.cs]: I just created an issue for adding more rules to the .editconfig.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-queue-max-size-to-reflect-namespace-sku.13.build.779 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37165160) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/2c8bc2b5ae by @SeanFeldman)

File: src/ServiceBusExplorer/Controls/HandleTopicControl.cs [hunk 2]
```diff
@@ -40,6 +40,8 @@
 
 namespace ServiceBusExplorer.Controls
 {
+    using ServiceBus.Helpers;
```
Comment (ErikMogensen) [inline src/ServiceBusExplorer/Controls/HandleTopicControl.cs:43]: That lonely using again.

Comment (SeanFeldman) [reply src/ServiceBusExplorer/Controls/HandleTopicControl.cs]: done

**Comment by SeanFeldman** [comment]
:x: [Build ServiceBusExplorer 5.1.0-queue-max-size-to-reflect-namespace-sku.14.build.781 failed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37166035) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/f56e411f50 by @SeanFeldman)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-queue-max-size-to-reflect-namespace-sku.14.build.783 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37166074) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/7b27b21608 by @SeanFeldman)

**Review by ErikMogensen — APPROVED** [review]
Happy to see that the new SDK is put to use. 

I reopened the old #99. It will be fixed by this PR.

**Review by paolosalvatori — APPROVED** [review]

**Comment by paolosalvatori** [comment]
Thanks @SeanFeldman for being so quick in implementing this feature request :)

**Comment by SeanFeldman** [comment]
Thank you, @paolosalvatori.
Another evidence of why I shouldn’t be touching UI 😀

**Comment by paolosalvatori** [comment]
Well, happy birthday @SeanFeldman. Usually, people use Facebook to wish happy birthday to friends, but you deactivated posts from others on your Facebook account :)

**Comment by SeanFeldman** [comment]
GitHub. This is the way!

Thank you @paolosalvatori 🙂

**Comment by ErikMogensen** [comment]
Happy birthday @SeanFeldman!  ✨ 🌟 💥 🍰 🎂 ⚡ 

Is there a better way to celebrate it than having gotten a PR merged into ServiceBusExplorer? 😄

**Comment by ErikMogensen** [comment]
Also fixes #99

**Comment by SeanFeldman** [comment]
Thank you, @ErikMogensen 🙂

> Is there a better way to celebrate it than having gotten a PR merged into ServiceBusExplorer? 😄

Yes! Have no follow-up issues raised 😂

**Pull request merged**